app/src/main/java/org/wikipedia/analytics/eventplatform/MachineGeneratedArticleDescriptionsAnalyticsHelper.kt

```diff
@@ -24,11 +25,20 @@ object MachineGeneratedArticleDescriptionsAnalyticsHelper {
         )
     }
 
+    fun logActualPublishedDescription(context: Context, publishedDescription: String, wikidataEdit: Boolean = false) {
+        EventPlatformClient.submit(
+            BreadCrumbLogEvent(
+                BreadCrumbViewUtil.getReadableScreenName(context),
+                "$MACHINE_GEN_DESC_SUGGESTIONS.publishedTo.${if (wikidataEdit) "wikidata" else "articleOnWiki"}publishedDescription.$publishedDescription.published"
```

Comment:
A couple of questions:
* Instead of logging "wikidata" versus "articleOnWiki", can we just log the wiki language code of the article? We know that only `en` articles are written to the wiki, and we'll get the bonus data of the language code.
* In this string I see three separate mentions of `publishedTo`, `publishedDescription`, and `published`. Do we need all three?
* Any reason not to include the article title in the event?

Reply:
Good idea @dbrant .. which also made me look into the fact that Shay will not be receiving article wiki anywhere in the event, just the primary wiki of the app. So added that information as well into the events that needed the info. Also named the `publish` related infos better to remove confusion.